modules/cluster-logging-collector-log-forward-es.adoc

```diff
@@ -91,3 +92,52 @@ The Red Hat OpenShift Logging Operator redeploys the Fluentd pods. If the pods d
 ----
 $ oc delete pod --selector logging-infra=fluentd
 ----
+
+.Example: Setting a secret that contains a username and password
+
+You can use a secret that contains a username and password to authenticate a secure connection to an external Elasticsearch instance.
+
+For example, if you cannot use mutual TLS (mTLS) keys because a third party operates the Elasticsearch instance, you can use HTTP or HTTPS and set a secret that contains the username and password.
+
+. Create a `Secret` YAML file similar to the following example that contains base64 encoded values for `username` and `password`:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: openshift-test-secret
+data:
+  username: dGVzdHVzZXJuYW1lCg==
+  password: dGVzdHBhc3N3b3JkCg==
+----
+
+. Create the secret:
++
+[source,terminal]
+----
+$ oc create secret -n openshift-logging openshift-test-secret.yaml
+----
+
+
+. Specify the name of the secret in your `ClusterLogForwarder`:
++
+[source,yaml]
+----
+kind: ClusterLogForwarder
+metadata:
+  name: instance
+  namespace: openshift-logging
+spec:
+  outputs:
+   - name: elasticsearch
+     type: "elasticsearch"
+     url: https://elasticsearch.secure.com:9200
+     secret:
+        name: openshift-test-secret
+----
+
+[NOTE]
+====
+With this type of secret, the prefix of the `url` value can be `http` or `https`.
+====
```

Comment:
I'd maybe move this to directly below the YAML for the secret itself?

Reply:
I think this location is best. This note is intended to help users understand the `url` field in the snippet that precedes the note. I don't want them to infer from this snippet that the value of `url` must only be `https...`  Perhaps there's a better way to convey this. What do you think?

Reply:
Maybe just a call-out either, but I see what you mean now on second look, so I think it's fine either way 🙂